docs/glossary.rst

```diff
@@ -207,13 +296,27 @@ Glossary
         where :math:`\pi` is a :term:`coupling` density with first marginal
         :math:`\mu` and second marginal :math:`\nu`.
 
+    Legendre transform
```

Comment:
Consider linking to this in the `entropy-regularized optimal` entry.

Reply:
nice catch!